alacritty_terminal/src/event.rs

```diff
@@ -1,21 +1,55 @@
+// Copyright 2020 Christian Duerr, The Alacritty Project Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 use std::borrow::Cow;
 use std::path::PathBuf;
 
+use crate::clipboard::ClipboardType;
 use crate::message_bar::Message;
 use crate::term::SizeInfo;
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone)]
 pub enum Event {
     DPRChanged(f64, (u32, u32)),
     ConfigReload(PathBuf),
     MouseCursorDirty,
     Message(Message),
     Title(String),
+    ClipboardStore(ClipboardType, String),
+    ClipboardLoad(ClipboardType, std::sync::Arc<dyn Fn(&str) -> String + Sync + Send + 'static>),
     Wakeup,
     Urgent,
     Exit,
 }
 
+impl std::fmt::Debug for Event {
```

Comment:
```suggestion
impl Debug for Event {
```

Import Debug at the top?